subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -158,9 +158,8 @@ static void lp_comm_tx(struct ll_conn *conn, struct proc_ctx *ctx)
 	/* Enqueue LL Control PDU towards LLL */
 	llcp_tx_enqueue(conn, tx);
 
-	/* Update procedure timeout. For TERMINATE supervision_timeout is used */
-	ull_conn_prt_reload(conn, (ctx->proc != PROC_TERMINATE) ? conn->procedure_reload :
-									conn->supervision_reload);
+	/* Restart procedure response timeout timer */
+	llcp_lr_prt_restart(conn);
```

Comment:
Before we had a distinction re. TERMINATE procedure and prt vs. sto - this is removed?

Reply:
Yes this is removed as I believe this i a misuse of the PRT timer.

BLUETOOTH CORE SPECIFICATION Version 5.3 | Vol 6, Part B, 5.1.6 ACL Termination procedure

> The Link Layer shall start a timer, Tterminate, when the LL_TERMINATE_IND
> PDU has been queued for transmission. The initiating Link Layer shall send
> LL_TERMINATE_IND PDUs until an acknowledgment is received or until the
> timer, Tterminate, expires, after which it shall exit the Connection State and
> transition to the Standby State. The initial value for Tterminate shall be set to the
> value of the connSupervisionTimeout.

> The procedure has completed when the acknowledgment has been received or
> the timer, Tterminate, expires.

When the terminate timer expires the procedure completes, but by using the PRT timer the HCI_Disconnection_Complete event will use _LL RESPONSE TIMEOUT (0x22)_ as reason and not the reason given in the HCI_Disconnect command.

The Tterminate timer will need to be handled in a separate PR.